### DISCUSSION TITLE
mrouted doesn't allow routing same group from 2 origins

### DISCUSSION THREAD

**Discussion by lucdig** [troglobit/mrouted#62] [Q&A]
Hi, I have this /etc/mrouted.conf:

```
no phyint
phyint tun0 enable static-group 239.13.13.0
phyint tun1 enable static-group 239.13.13.0
```

1) When a packet originated from tun0 (10.13.13.10) is sent to 239.13.13.0, it is routed to tun1.
2) But, when a packet from tun1 (10.13.13.11) is sent to 239.13.13.0, it is not routed to tun0.

and viceversa.

2) works after 2-3 minutes, maybe when a cache's timer expires.

after 1) I see:
```
Multicast Forwarding Cache Table
Origin          Group           Inbound       <>    Uptime   Expire  Outbound
10.13.13.10/30   239.13.13.0     tun0               0:08:26  0:02:28   tun1
10.13.13.11/30   239.13.13.0     tun1               0:10:57  0:02:09   tun0

Source          Group           Inbound             Uptime  Packets     Bytes
10.13.13.10      239.13.13.0     tun0               0:00:49
```

I'm running mrouted on OpenBSD 7.5

Thanks for any help

**Comment by lucdig** [comment]
Ruuning mrouted at foreground, I see this after a few time, don't know what it could mean:

`mrouted: 13:27:36.471 warning - Failed reading (S,G) count for (10.13.13.11,239.13.13.0): Invalid argument`

**Reply by troglobit** [reply]
OpenBSD has their own in-tree version of mrouted, is that what you are using?

**Reply by lucdig** [reply]
I downloaded and compiled the 4.5 version, I'm not using the in-tree version

**Reply by troglobit** [reply]
OK. It's probably best if you use a native OpenBSD service for your needs since I've not tested my fork on the BSDs for many years now.
